var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -33,6 +33,7 @@ class Fftw(AutotoolsPackage):
     patch('pfft-3.3.5.patch', when="@3.3.5:+pfft_patches", level=0)
     patch('pfft-3.3.4.patch', when="@3.3.4+pfft_patches", level=0)
     patch('pgi-3.3.6-pl2.patch', when="@3.3.6-pl2%pgi", level=0)
+    patch('a64fx-3.3.8.patch', when="@3.3.8 target=a64fx")
```

Comment:
Can you provide a reference where this patch is discussed? Asking since the patch changes ~8k lines in the sources. Will it be accepted upstream for future releases?

Reply:
FWIW, the patch is quite big because it modifies a bunch of files that are automatically generated, so it would be much easier to review the patch vs the FFTW/fftw3 repo.
That being said, such a patch would not be a fit for spack since the FFTW `bootstrap.sh` is a dependency hell (including an old `ocaml` version that supports a removed module) and spack should keep building fftw from a tarball.

Reply:
> FWIW, the patch is quite big because it modifies a bunch of files that are automatically generated, so it would be much easier to review the patch vs the FFTW/fftw3 repo

I concur, my question was more like to know if such a PR was submitted upstream to the FFTW repo and if it's worth to get it merged there and then use the latest release, in particular considering that it's likely that a new release will be tagged soon: https://github.com/FFTW/fftw3/issues/210